makefile.shade

```diff
@@ -614,6 +621,20 @@ var CAN_BUILD_ONECORE = '${Directory.Exists(VS_ONECORE_ROOT) && Directory.GetFil
                     Log.Info("Copying " + file + " to " + targetFilePath);
                     File.Copy(file, targetFilePath, true);
                 }
+                
+                if (target.OS != "win")
+                {
+                    foreach (var native in nativeLibraries)
+                    {
+                        var targetFilePath = Path.Combine(target.TargetFolder, "lib", "Microsoft.Dnx.Tooling", native + nativeExtension);
+                        var sourceFile = Path.Combine(target.TargetFolder, native + nativeExtension);
+                        if (File.Exists(sourceFile))
+                        {
+                            Log.Info("Moving " + native + " to " + targetFilePath);
+                            File.Move(Path.Combine(target.TargetFolder, native + nativeExtension), targetFilePath);
```

Comment:
Yep, there isn't a problem, just can replace some of the stuff with the variable to make it look nicer.